apiserver/backup.go

```diff
@@ -12,20 +12,12 @@ import (
 	"github.com/juju/errors"
 
 	apiservererrors "github.com/juju/juju/apiserver/errors"
-	apiserverbackups "github.com/juju/juju/apiserver/facades/client/backups"
-	"github.com/juju/juju/apiserver/httpattachment"
 	"github.com/juju/juju/apiserver/params"
-	"github.com/juju/juju/state"
 	"github.com/juju/juju/state/backups"
 )
 
-var newBackups = func(st *state.State, m *state.Model) (backups.Backups, io.Closer) {
-	backend := struct {
-		*state.State
-		*state.Model
-	}{st, m}
-	stor := backups.NewStorage(backend)
-	return backups.NewBackups(stor), stor
+var newBackups = func() backups.Backups {
+	return backups.NewBackups()
```

Comment:
what about `newBackups := backups.Newbackups`

Reply:
yeah good point, the func was needed before but not now